.goreleaser.yaml

```diff
@@ -106,6 +106,10 @@ dockers:
       - "--build-arg=BUILD_VERSION={{.Version}}"
       - "--build-arg=VCS_REF={{.FullCommit}}"
       - "--build-arg=VCS_URL={{.GitURL}}"
+      - "--label=io.artifacthub.package.readme-url=https://raw.githubusercontent.com/anchore/grype/main/README.md"
+      - "--label=io.artifacthub.package.logo-url=https://user-images.githubusercontent.com/5199289/136855393-d0a9eef9-ccf1-4e2b-9d7c-7aad16a567e5.png"
+      - "--label=io.artifacthub.package.license=Apache-2.0"
+      - "--label=org.opencontainers.image.description=A vulnerability scanner for container images and filesystems"
```

Comment:
Probably link to https://github.com/anchore/grype#readme ?

Reply:
Oh, that's interesting so reads GitHub flavored markdown?

Reply:
I think so, yes 🙋🏻‍♂️

Reply:
Because, @dirien have been worked for the GoReleaser project for the same purpose, see.
https://github.com/goreleaser/goreleaser/issues/2830
https://artifacthub.io/packages/container/goreleaser/goreleaser

Reply:
Yep, thats true @developer-guy!